### ISSUE TITLE
Error is raised for problematic rows

### ISSUE THREAD

**Issue by TomerAntman** [OpenGene/fastp#560]
In some of the fastq files that I have, there are some problematic rows, such as this:
```
AAAAAEEEE...
@NB552437:11:HY5W2BGXG:1:11101:12897:1306 1:N:0:GAATTCGT+GTCAGTAC

+

@NB552437:11:HY5W2BGXG:1:11101:22688:1308 1:N:0:GAATTCGT+GTCAGTAC
CAGCGAGGG...
+
AAAAAEEEE...
@NB552437:11:HY5W2BGXG:1:11101:9713:1313 1:N:0:GAATTCGT+GTCAGTAC
TTTTCTTGA...
```
These problematic rows crash the code and the error is:
> Expected '+', got 
> ERROR: '+' expected

I tried just deleting the empty rows but it still doesn't work.
Trimmomatic seems to be able to bypass this issue (it doesn't crash)

**Comment by nickgladman** [comment]
Hello. I am having the same problem, which seems to be identical to #534. 
`"Expected '+', got CCACAGTGAGCCGCCGATTTGTTCCATGCCTGCAC...`

It seems like it causes the run to terminate at the problematic read rather than continuing. @TomerAntman are the quality threshold parameters you're using for trimmomatic the same for the fastp run?

**Comment by TomerAntman** [comment]
@nickgladman yes they are as far as I remember. I solved this the other day by running `seqkit sana` on the files and then `seqkit pair`. It solved the issue for most of my files

**Comment by sfchen** [comment]
Could you please upload a piece of your data so that I can reproduce this issue?
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Sun, Jun 2, 2024 03:04 AM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; 
Subject: &nbsp;Re: [OpenGene/fastp] Error is raised for problematic rows (Issue #560)

&nbsp;



 
Hello. I am having the same problem, which seems to be identical to #534.
 "Expected '+', got CCACAGTGAGCCGCCGATTTGTTCCATGCCTGCAC...
 
It seems like it causes the run to terminate at the problematic read rather than continuing. @TomerAntman are the quality threshold parameters you're using for trimmomatic the same for the fastp run?

**Comment by sfchen** [comment]
fastp should already support these kind of blank reads. If you still observe this error, can you please upload a piece of data so that I can reproduce this issue?
